xdsl/dialects/arith.py

```diff
@@ -470,6 +470,9 @@ def get(operand1: Union[Operation, SSAValue],
             result_types=[IntegerType.from_width(1)],
             attributes={"predicate": IntegerAttr.from_int_and_width(arg, 64)})
 
+    def verify_(self) -> None:
```

Comment:
This overriding is actually not needed, right? We are not enforcing anything in this constraint, so I would leave this verifier out.

If you want to fill it, you could for example check that predicate is between 0 and 9.

Reply:
agree